server/chat-plugins/wifi.tsx

```diff
@@ -1335,34 +1401,36 @@ export const commands: Chat.ChatCommands = {
 		this.runBroadcast();
 		const buf = [];
 		if (user.can('show', null, room)) {
-			buf.push(
-				`<details><summary>Staff commands</summary>` +
-				`<code>/giveaway create</code> - Pulls up a page to create a giveaway. Requires: + % @ # &`
-			);
-			buf.push(
-				`<code>/giveaway create question Giver | OT | TID | Game | Question | Answer 1, Answer 2, Answer 3 | IV/IV/IV/IV/IV/IV | Pok&eacute; Ball | Extra Info | Prize</code>` +
-				` - Start a new question giveaway (voices can only host their own). Requires: + % @ # &`,
-			);
-			buf.push(
-				`<code>/giveaway create lottery Giver | OT | TID | Game | # of Winners | IV/IV/IV/IV/IV/IV | Pok&eacute; Ball | Extra Info | Prize</code>` +
-				` - Start a new lottery giveaway (voices can only host their own). Requires: + % @ # &`,
-			);
-			buf.push(`<code>/giveaway changequestion/changeanswer</code> - Changes the question/answer of a question giveaway. Requires: Being giveaway host`);
-			buf.push(`<code>/giveaway viewanswer</code> - Shows the answer of a question giveaway. Requires: Being giveaway host/giver`);
-			buf.push(`<code>/giveaway ban [user], [reason]</code> - Temporarily bans [user] from entering giveaways. Requires: % @ # &`);
-			buf.push(`<code>/giveaway end</code> - Forcibly ends the current giveaway. Requires: % @ # &`);
-			buf.push(`<code>/giveaway count [pokemon]</code> - Shows how frequently a certain Pok&eacute;mon has been given away.</details>`);
+			buf.push(<details><summary>Staff commands</summary>
+				<code>/giveaway create</code> - Pulls up a page to create a giveaway. Requires: + % @ # &amp;
+				<br /><code>
```

Comment:
We tend to end lines with `<br />` rather than starting them.

Reply:
I’ll fix this, but this is done so inconsistently throughout the sim that there was no way to know this.

Reply:
That's very unfortunate. I've always broken lines after `<br />`. Maybe I'll need to add it to the style guide.